gravitee-apim-console-webui/src/components/notifications/notifications-list/notification-details/notification-settings-details.component.html

```diff
@@ -23,11 +23,20 @@
   </div>
   <form *ngIf="!isLoadingData" autocomplete="off" gioFormFocusInvalid [formGroup]="notificationForm" (ngSubmit)="onSubmit()">
     <mat-card>
-      <mat-form-field class="notifications-form__email" *ngIf="notifier">
-        <mat-label>Email list</mat-label>
-        <input matInput formControlName="notifier" />
-        <mat-hint>Use space, ',' or ';' to separate emails. EL supported.</mat-hint>
-      </mat-form-field>
+      <ng-container *ngIf="notifier">
+        <mat-form-field class="notifications-form__notifier">
+          <mat-label *ngIf="notifier.type === 'EMAIL'">Email list</mat-label>
+          <mat-label *ngIf="notifier.type === 'WEBHOOK'">Webhook</mat-label>
+          <input matInput formControlName="notifier" />
+          <mat-hint *ngIf="notifier.type === 'EMAIL'">Use space, ',' or ';' to separate emails. EL supported.</mat-hint>
+          <mat-hint *ngIf="notifier.type === 'WEBHOOK'">URL (Gravitee will POST datas to this url)</mat-hint>
+        </mat-form-field>
+        <div *ngIf="notifier.type === 'WEBHOOK'" class="checkbox-use-system-proxy">
+          <mat-checkbox formControlName="useSystemProxy">
```

Comment:
Could you use mat-toggle instead of mat-checkbox. The UI/Ux team prefer the use of toggle ;)

Reply:
you can found some example on already migrated pages

Reply:
Done